lib/workers/branch/schedule.js

```diff
@@ -65,14 +77,15 @@ function isScheduledNow(config) {
     logger.warn(errorMessage);
     return true;
   }
+  const [validTimezone, error] = hasValidTimezone(config.timezone);
```

Comment:
This should only be done if a timezone is configured. Move back inside the if that was removed